model/model.go

```diff
@@ -218,6 +218,9 @@ type TableInfo struct {
 	// The pre-split region num is 2^(PreSplitRegions-1).
 	// And the PreSplitRegions should less than or equal to ShardRowIDBits.
 	PreSplitRegions uint64 `json:"pre_split_regions"`
+	// WaitSplitFinish specify the pre-split region to do with synchronous or asynchronous.
+	// false means pre-split region with asynchronous.
+	WaitSplitFinish bool `json:"wait_split_finish"`
```

Comment:
We can put this option as a session variable, it is weird to be a table option. And we should not add this field in the tableInfo.

Reply:
em... @morgo, Do you prefer session variable or table option to control synchronously/asynchronism split?

Reply:
+1 for session var